lib/verk/workers_manager.ex

```diff
@@ -193,6 +193,8 @@ defmodule Verk.WorkersManager do
         Log.start(job, worker)
         Verk.Worker.perform_async(worker, self, job)
         notify!(%Events.JobStarted{job: job, started_at: Timex.DateTime.now})
+      :full ->
+        Verk.enqueue(job)
```

Comment:
should I remove this clause?

This should be unnecessary now that poolboy is used to determine how many available workers there are?